.github/workflows/build.yml

```diff
@@ -67,25 +80,33 @@ permissions:
 
 env:
   # include commit in path in case release commit gets updates on subsequent runs
-  s3_bucket: "s3://${{ inputs.s3_bucket_name }}/${{ github.repository }}/${{ inputs.version_number }}/${{ inputs.sha }}"
-  artifact_retention_days: 2
+  AWS_S3_BUCKET: "s3://${{ inputs.s3_bucket_name }}/${{ github.repository }}/${{ inputs.version_number }}/${{ inputs.sha }}"
```

Comment:
As discussed in slack, we should use 2 directories within the same bucket, one for test runs and one for regular runs.

Reply:
Looks like we no longer need this environment variable then.

Reply:
> Looks like we no longer need this environment variable then.

Yup, removed in https://github.com/dbt-labs/dbt-release/pull/37/commits/4e112a7802e6b89d9f55dbc39d7f5f17feb31baf